.github/workflows/review.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Add review url
+
+on:
+  pull_request_target:
```

Comment:
Should this have been `pull_request` as https://github.com/ome/ngff/pull/112#issuecomment-1141013652 seems to link to the last commit on the base branch as per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target?

Reply:
Grrr. The actions can't write to the same PR if only `pull_request` is used. I'll have to look for a way to find the sha regardless. Let me know if we need to revert.

cc: @jburel in case you've run into these issues before.